feature_creation_scripts/ismip6_antarctic_ocean_regions/ismip6_antarctic_ocean_regions.py

```diff
@@ -0,0 +1,279 @@
+
+    def stereo_to_lon_lat(x, y):
+        return pyproj.transform(inProj, outProj, x, y)
+
+    ds = xarray.open_dataset('bedmap2.nc')
+
+    # plot contours
+    plt.figure()
+    cs = plt.contour(ds.x.values, ds.y.values, ds.bed, (contourValue,))
+    paths = cs.collections[0].get_paths()
+
+    pathLengths = [len(paths[i]) for i in range(len(paths))]
+    iLongest = numpy.argmax(pathLengths)
+
+    p = paths[iLongest]
+    v = p.vertices
+    x = v[:, 0]
+    y = v[:, 1]
+
+    # Antarctic stereographic
+    inProj = pyproj.Proj(init='epsg:3031')
+    # lon/lat
+    outProj = pyproj.Proj(init='epsg:4326')
+
+    poly = shapely.geometry.Polygon([(i[0], i[1]) for i in zip(x, y)])
+
+    epsilon = 1e-14
+    minY = numpy.amin(y)
+    wedge = shapely.geometry.Polygon([(epsilon, minY),
+                                      (epsilon**2, -epsilon),
+                                      (0, epsilon),
+                                      (-epsilon**2, -epsilon),
+                                      (-epsilon, minY),
+                                      (epsilon, minY)])
+
+    difference = poly.difference(wedge)
+
+    difference = shapely.ops.transform(stereo_to_lon_lat, difference)
+
+    fc = FeatureCollection()
+
+    geometry = shapely.geometry.mapping(difference)
+    # get rid of the wedge again by rounding the coordinates
+    geometry['coordinates'] = _round_coords(geometry['coordinates'])
+
+    fc.add_feature(
+        {"type": "Feature",
+         "properties": {"name": "Contour {}".format(contourValue),
+                        "author": author,
+                        "object": 'region',
+                        "component": 'ocean'},
+         "geometry": geometry})
+
+    return fc
+
+
+def make_polygon(lons, lats, name, author, tags):
+    fc = FeatureCollection()
+
+    coords = list()
+    for index in range(len(lons)):
+        coords.append([lons[index], lats[index]])
+    coords.append([lons[0], lats[0]])
+
+    fc.add_feature(
+        {"type": "Feature",
+         "properties": {"name": name,
+                        "author": author,
+                        "object": 'region',
+                        "component": 'ocean',
+                        "tags": tags,
+                        "zmin": -1000.,
+                        "zmax": -400.},
+         "geometry": {
+             "type": "Polygon",
+             "coordinates": [coords]}})
+    return fc
+
+
+def shelf_polygon(lons, lats, name, author, tags, fcContour):
+    fc = make_polygon(lons, lats, name, author, tags)
+
+    lons = [-180., -180., 180., 180.]
+    lats = [-90., 90., 90., -90.]
+    fc_world = make_polygon(lons, lats, name, author, tags)
+
+    fcContour = fc_world.difference(fcContour)
+
+    fcShelf = fc.difference(fcContour)
+
+    props = fcShelf.features[0]['properties']
+    props['name'] = props['name'] + ' Shelf'
+    props['tags'] = props['tags'] + ';Shelf'
+    props['zmin'] = -1000.
+    props['zmax'] = -200.
```

Comment:
Hes'e where we need to set the `zmin` and `zmax` we want.  Of course, we can always do something else in MPAS-Analysis but we might as well start with something useful.

Reply:
Thanks Xylar. The depth range we are looking for is 1500-200, so zmax is ok but zmin should be changed to -1500. Can you confirm the regions were set with a 1500m depth threshold? Both the averaging and the spatial regions have a 1500m limit.

Reply:
Okay, great, I'll make that change.